src/main/java/bobcat/exception/BobCatException.java

```diff
@@ -0,0 +1,7 @@
+package bobcat.exception;
+
+public class BobCatException extends RuntimeException {
```

Comment:
Perhaps you would like to include a header comment for this class since it is public?

Reply:
Noticed this in several other places too.